accounts/models.py

```diff
@@ -65,6 +72,9 @@ def withdraw(self, amount: decimal.Decimal) -> None:
         if type(amount) is not decimal.Decimal:
             amount: decimal.Decimal = decimal.Decimal(amount)
 
+        if amount < 0:
```

Comment:
Same suggestion as mentioned before about convert integer value into decimal value.

```suggestion
        if amount < decimal.Decimal(0.0):
```